official/resnet/keras/keras_cifar_main.py

```diff
@@ -18,7 +18,6 @@
 from __future__ import division
 from __future__ import print_function
 
-from absl import app as absl_app
```

Comment:
This is needed on line 209.

Reply:
Ah I see! Sorry I was following the `g4 lint` outputs and didn't notice the rewrite in OSS.